scripts/deploySidechain.ts

```diff
@@ -88,116 +91,121 @@ export async function deploySidechainSystem(
     waitForBlocks: number = 0,
 ): Promise<SidechainDeployed> {
     const deployerAddress = await deployer.getAddress();
+    const create2Options = { amount: 0, salt: undefined, callbacks: [] };
+    const deployOptions = {
+        overrides: {},
+        create2Options,
+        debug,
+        waitForBlocks,
+    };
+
     const create2Factory = Create2Factory__factory.connect(addresses.create2Factory, deployer);
+
+    const voterProxySetOwner = VoterProxyLite__factory.createInterface().encodeFunctionData("setOwner", [
+        deployerAddress,
+    ]);
+
     const voterProxy = await deployContractWithCreate2<VoterProxyLite, VoterProxyLite__factory>(
+        hre,
         create2Factory,
         new VoterProxyLite__factory(deployer),
         "VoterProxyLite",
-        [addresses.minter, addresses.token, deployerAddress],
-        {},
-        {},
-        debug,
-        waitForBlocks,
+        [addresses.minter, addresses.token],
+        { ...deployOptions, create2Options: { ...create2Options, callbacks: [voterProxySetOwner] } },
     );
-    // Ownable
+
+    const coordinatorTransferOwnership = Coordinator__factory.createInterface().encodeFunctionData(
+        "transferOwnership",
+        [deployerAddress],
+    );
+
     const coordinator = await deployContractWithCreate2<Coordinator, Coordinator__factory>(
+        hre,
         create2Factory,
         new Coordinator__factory(deployer),
         "Coordinator",
-        [
-            naming.coordinatorName,
-            naming.coordinatorSymbol,
-            addresses.lzEndpoint,
-            extConfig.canonicalChainId,
-            deployerAddress,
-        ],
-        {},
-        {},
-        debug,
-        waitForBlocks,
+        [naming.coordinatorName, naming.coordinatorSymbol, addresses.lzEndpoint, extConfig.canonicalChainId],
+        { ...deployOptions, create2Options: { ...create2Options, callbacks: [coordinatorTransferOwnership] } },
     );
 
     const cvxTokenAddress = coordinator.address;
+    const boosterSetOwner = BoosterLite__factory.createInterface().encodeFunctionData("setOwner", [deployerAddress]);
```

Comment:
I think we just do the ownership transfer via execute. The other functions can be called as normal in this script